src/app/markdown/markdown.component.less

```diff
@@ -9,13 +9,15 @@
   width: 100%;
   background-color: @color-pf-black-200;
   .editor {
+    word-wrap: break-word;
     .editor-markdown {
       font-size: 1.071em;
         margin: 0;
     }
     .editor-preview {
-      h1 {
+      h1, h2, h3, h4, h5, h6 {
```

Comment:
I thought @mindreeper2420 said we were not supposed to use H1 or the others. @dlabrecq @SMahil do you remember?

Reply:
I think this is okay as it's a special case where it applies the styles in markdown-html element where you can't define a class.

Reply:
I don't think we should use heading tags here. If in any case we can't define classes we can put that tag into hierarchies and imply the styles

Reply:
What do you mean by hierarchy? It is in the maximum hierarchy position we can detect. Anything under editor-preview is server generated.

Reply:
@SMahil It is in hierarchal form  ```.editor-container > .editor.editor-preview > h1, h2, h3, h4, h5, h6```